EventsExpress/ClientApp/src/components/event/event-item-view.js

```diff
@@ -159,11 +242,70 @@ export default class EventItemView extends Component {
                         }
                     </div>
                     <div className="col-3 overflow-auto shadow p-3 mb-5 bg-white rounded">
-                        {this.renderOwner(user)}
-                        {this.renderUsers(visitors)}
+                        <ExpansionPanel>
```

Comment:
Let's extract how visitors are dsiplayed and logic of maintenance into separate component and render it here.